docs/Trust Registry.md

```diff
@@ -56,3 +57,136 @@ where `"z5Bug71M7Sj7cYpbVBDmN:2:test_schema:0.3"` represents the schema ID, name
 > **_NOTE_**: In a production environment, this should not be exposed to the internet or interacted with directly.
 > It's advisable to either avoid exposing this to the internet or set up a separate security layer for the trust
 > registry. This is because it's crucial to prevent unauthorized individuals from making changes to the trust registry.
+
+## Trust-registry Role in application flows
```

Comment:
"Trust Registry" not "Trust-registry". A few cases to replace in following lines.

This heading can be "Trust Registry Interactions During Application Flows"